Sources/ArcGISToolkit/Components/Bookmarks/Bookmarks.swift

```diff
@@ -77,74 +62,74 @@ public struct Bookmarks: View {
     /// - Parameters:
     ///   - isPresented: Determines if the bookmarks list is presented.
     ///   - bookmarks: An array of bookmarks. Use this when displaying bookmarks defined at runtime.
-    ///   - viewpoint: A viewpoint binding that will be updated when a bookmark is selected.
-    ///   Alternately, you can use the `onSelectionChanged(perform:)` modifier to handle
-    ///   bookmark selection.
+    ///   - selection: A selected Bookmark.
+    ///   - geoViewProxy: The proxy to provide access to geo view operations.
+    ///
+    /// When a `GeoViewProxy` is provided, the map or scene will automatically pan and zoom to the
+    /// selected bookmark.
     public init(
```

Comment:
If you move all of these into an extension, then the compiler would synthesize a memberwise initializer that the other initializers could delegate to:

![image](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/assets/2257493/370233c3-b07f-4ab3-a858-f426426544ea)

For example:

```swift
public extension Bookmarks {
    /// ...
    init(
        isPresented: Binding<Bool>,
        bookmarks: [Bookmark],
        selection: Binding<Bookmark?>,
        geoViewProxy: GeoViewProxy? = nil
    ) {
        self.init(
            bookmarkSource: .array(bookmarks),
            geoViewProxy: geoViewProxy,
            isPresented: isPresented,
            selection: selection
        )
    }
    
    ...
}
```

I suggest doing that.